Assets/Scripts/Combat/MusicManager.cs

```diff
@@ -14,34 +15,49 @@ public class MusicManager : MonoBehaviour
 
     void Start()
     {
+        aud[0] = gameObject.AddComponent<AudioSource>();
+        aud[1] = gameObject.AddComponent<AudioSource>();
         song = Random.Range(0, CombatSongs.Length);
     }
 
     public void SetVictory()
     {
         if (gameOver) return;
-        AudioSource thisSource = GetComponent<AudioSource>();
-        thisSource.clip = VictorySong;
-        thisSource.Play();
+        StartCoroutine(crossFade(1.0f, VictorySong));
         gameOver = true;
     }
 
     public void SetDefeat()
     {
         if (gameOver) return;
-        AudioSource thisSource = GetComponent<AudioSource>();
-        thisSource.clip = DefeatSong;
-        thisSource.Play();
+        StartCoroutine(crossFade(1.0f, DefeatSong));
         gameOver = true;
     }
 
+    IEnumerator crossFade(float duration, AudioClip fadeTo)
```

Comment:
Rename the method to CrossFade (first letter capitalization).

Reply:
Good call, thanks!